src/scenes/home/faq/questions.js

```diff
@@ -0,0 +1,117 @@
+    },
+    {
+      question: 'What is available to start learning to code today?',
+      answer:
+        "Our friends at the New York City-based, The Flatiron School created <a href='https://learn.co/'>Learn.co</a>, an online platform to get introduced to web development and the popular web framework, Ruby on Rails. Request an invite <a href='https://learn.co/join/operation-code'>here</a> and then join the #learn-dot-co channel in our Slack. Another resource is <a href='https://www.learnhowtoprogram.com/courses'>learnhowtoprogram.com</a>, a resource maintained by Epicodus."
+    },
+    {
+      question: 'What are the hours of operation for Operation Code?',
+      answer:
+        "Operation Code is different in that we don't have regular business office hours. The team can usually be found in our <a href='http://operation-code.slack.com'>Slack channel</a>, or on <a href='https://github.com/OperationCode/operationcode'>GitHub</a> fixing bugs and implementing new features."
+    },
+    {
+      question: "How can I help, if I can't afford to donate to Operation Code?",
+      answer:
+        "In addition to requiring financial support, we also need <a href='http://op.co.de/volunteer' target='_blank'>volunteers</a> and interns. The larger our community, the more we can spread the word about our work. Also, remember that every <a href='https://secure.lglforms.com/form_engine/s/BRtP7QUKyHOyEYsZROsRew'>donation</a>, no matter how modest, brings us closer to our goals."
+    },
+    {
+      question:
+        'I would like to receive Operation Code updates and news. How can I receive these communications?',
+      answer:
+        "We primarily use <a href='https://twitter.com/operation_code'>Twitter</a> and <a href='http://facebook.com/operationcode.org'>Facebook</a> to put out updates and news since it's faster to put out info and respond. Given our chosen craft, we don't do regular emails as often."
+    },
+    {
+      question: "My question isn't listed. How do I contact Operation Code?",
+      answer:
+        "If you have a question that isn't listed here on our FAQ or our <a href='https://medium.com/@operation_code'>blog</a>, write to <a href='mailto:staff@operationcode.org'>staff@operationcode.org</a> and we'll get back to you as soon as we can."
+    }
+  ],
+  donation: [
+    {
+      question: 'What is the fastest way to make a donation?',
+      answer:
+        "The fastest way to make a donation is through our secured online form <a href='https://secure.lglforms.com/form_engine/s/BRtP7QUKyHOyEYsZROsRew'>here</a>."
+    },
```

Comment:
This URL should be the the imported `commonUrl.donateLink` instead of being hardcoded in this file.